README.md

```diff
@@ -3,147 +3,214 @@ cordova-plugin-xapkreader (Agamemnus/Flyingsoft Games edition)
 
-b) Add a meta tag to your index.html file (and perhaps other html files?): ``<meta http-equiv="Content-Security-Policy" content="* * 'self' default-src 'unsafe-inline' 'unsafe-eval' http://* https://* data: cdvfile://* content://*;">``
+[1]: https://developer.android.com/google/play/licensing/setting-up.html
+[2]: https://developer.android.com/guide/topics/providers/content-provider-basics.html#ContentURIs
 
-c) Add the following to ``[root]/config.xml``:
-````
-    <allow-navigation href="*://*/*"/>
-    <allow-intent href="*" />
-    <access origin="*" />
-    <access origin="content:///*" />
-    <access origin="cdvfile:///*" />
-````
 
-#Usage
+# Usage
+
+## Expansion files (OBB files)
+
+The expansion file should be a non-compressed ZIP file (also known as a "STOR" or "store"). That is, a ZIP file with 0% compression. Your ZIP file *can* include a directory structure, which can help to keep the files organized. 
+
+In Ubuntu Linux, you can generate a non-compressed zip file by adding the `-0` flag to the standard CLI `zip` command.
+
+In Windows, I use 7zip, which shows "version 20" in the file properties. (Some zip programs may generate zips that, when uploaded to the Google Play Developer Console, come back as corrupt OBBs...)
 
- Access your files via the content provider's expansion authority URI. E.G.:
+(You may ask, why use a ZIP file if it's not compressed? The reason is that Google requires us to use just one file, hence a ZIP. Leaving it uncompressed lets us quickly read from it, and since most media formats are already compressed, compression is unnecessary anyway.)
+
+Google doesn't care what name you give the file when you upload it. When the file is downloaded onto the user's device it will be [automatically renamed][3] to `[main|patch].<expansion-version>.<package-name>.obb]`; e.g.: `main.18009.org.example.mycordova.obb`. Hence, these are called **OBB files**.
+
+[3]: https://developer.android.com/google/play/expansion-files.html#Filename
+
+
+### Main & Patch files
+
+Google allows you to provide two expansion files, which it refers to as the `main` file and `patch` file. You're free to use these however you want, but the recommended pattern is that you start with a big `main` expansion file in your first release. Then, if there's a subsequent release where you need to change just one of the many files archived in `main`, you can upload a `patch` file that contains *only* that one changed file (and leave `main` unchanged). Then your users will have only that small download for their next upgrade, instead of having to download a whole big `main` file again.
+
+This plugin helps you to do that, by preferring files in `patch` over files in `main`. That is, if you have a file `pics/kitten.jpg` in your `main` and your `patch`, the plugin will use the `kitten.jpg` from `patch` and ignore the one in `main`.
+
+
+### Expansion file version numbers
```

Comment:
Do we need this section? The plugin already does all of this so isn't this step extraneous? Google Play should almost always (if not always) delete the old version.